### DISCUSSION TITLE
Tic Tac Vue - dare you?

### DISCUSSION THREAD

**Discussion by JoeldelPilar** [EyevinnOSC/community#41] [Show and tell]
Challenge your co-workers during break time? Try this fun little game now on Open Source Cloud! 😁

https://app.osaas.io/dashboard/service/joeldelpilar-tic-tac-vue

**Comment by anderswassen** [comment]
Nice game! Killed some free time using it :-)

Found a bug: If the players have the same name, eg Joel, the scoreboard will only show one name (Joel) with the combined score.
So either you have to chose different names or you are not allowed to play with friends that have the same name ;-)

Keep up the good work!

**Reply by JoeldelPilar** [reply]
Thank you for trying it out! 🤩  @anderswassen 

Thanks for reporting the bug, 🐛 for now we call it a feature 😆

**Reply by anderswassen** [reply]
Agreed, it's a feature!
